packages/dialtone-vue2/components/mode_island/mode_island.stories.js

```diff
@@ -0,0 +1,295 @@
+import DtModeIsland from './mode_island.vue';
+import ModeIslandDefaultStory from './mode_island_default.story.vue';
+import DtButton from '@/components/button/button.vue';
+import DtBadge from '@/components/badge/badge.vue';
+import DtChip from '@/components/chip/chip.vue';
+import DtInput from '@/components/input/input.vue';
+import DtNotice from '@/components/notice/notice.vue';
+import DtStack from '@/components/stack/stack.vue';
+import { createRenderConfig } from '@/common/storybook_utils';
+import { DT_MODE_ISLAND_TYPES } from './mode_island_constants';
+
+export const argsData = {
+  mode: DT_MODE_ISLAND_TYPES.INVERTED,
+  as: 'div',
+};
```

Comment:
I expect the storybook controls will work in the examples.
I can see there is some change in the code but it doesn't reflect in the examples.


https://github.com/user-attachments/assets/b58cf5b4-b6a4-4635-aa74-de2599197d25

Reply:
Done